frontend-react/src/pages/daily/Table/ReportsTable.tsx

```diff
@@ -80,24 +80,22 @@ function ReportsTable({ sortBy }: { sortBy?: string }) {
     };
 
     return (
-        <section className="grid-container margin-top-5">
-            <div className="grid-col-12">
-                {
-                    /* Button group only shows when there is more than a single sender. */
-                    receiverSVCs.length > 1 ? (
-                        <TableButtonGroup
-                            senders={receiverSVCs}
-                            chosenCallback={handleCallback}
-                        />
-                    ) : null
-                }
-                <Table config={resultsTableConfig} />
-                {reports.filter((report) => report.receivingOrgSvc === chosen)
-                    .length === 0 ? (
-                    <p>No results</p>
-                ) : null}
-            </div>
-        </section>
+        <div className="grid-col-12">
```

Comment:
If removing here, buttons will need to be bound another way. This makes it so there is no longer an outer most `grid-container` for this section. 
<img width="1775" alt="Screen Shot 2022-05-16 at 10 21 30 AM" src="https://user-images.githubusercontent.com/640182/168615542-a9d1b252-dc15-47a7-b5ad-320f0e6c1ac3.png">